packages/zoe/src/zoeService/internal-types.js

```diff
@@ -143,4 +144,20 @@
  */
 
 /**
+ * @typedef {Object} Meter
```

Comment:
```suggestion
 * @typedef {Object} Meter
 * All `bigint` units here are in computrons.
```

I think this is worth noting for clarity. I know I often go to our *types.js files first when trying to understand something. Reading ahead, there are a *lot* of places where I see a `bigint` and have to first as "Are these computrons?" I understand why we don't want runtime tagging of these. But just as a notational matter, how about we do

```js
/** @typedef {bigint} Computrons */
```

somewhere and then use it instead of `bigint` where appropriate, including the property declarations below?

Reply:
Tracking down where this `Meter` type is implemented, I see it in `vatAdminWrapper` within SwingSet, where its type is not declared. Should the type be declared there and then imported here from there? Which package in our system is best understood as defining the `Meter` type, as opposed to just implementing it?

Reply:
Yes, I think this type should really come from SwingSet.

Reply:
I'm happy to add one there and export it.. what's our best-practices for exporting externally-visible types from a package? Should I make a `packages/SwingSet/types/meter.js`? I imagine we want the `src/` to be hidden and eventually use the NESM `exports:` mapping in the `package.json` to conceal the internal location of this explicitly-exported name, yeah?

Reply:
I thought externally-visible types were supposed to go in a top-level `exported.js` file.

Reply:
Sounds good @warner. I agree with Chris that it should be included (possibly by importing from another file) in `exported.js`.